libopenage/gamestate/game_save.cpp

```diff
@@ -12,93 +14,242 @@
 #include "game_main.h"
 #include "game_save.h"
 #include "game_spec.h"
+#include "picojson.h"
```

Comment:
i think we can add it as an external dependency easily instead of bundling it. also avoids the licensing overhead and adding stuff to our copying.md